functions/v2/helloworld/hello_pubsub_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: New files use current year.
```suggestion
// Copyright 2021 Google LLC
```